swiftinterfaceparser/SwiftInterface.g4

```diff
@@ -297,7 +297,7 @@ dyckSubExpression :
 	| any_other_things_for_dyck_expression;
 
 any_other_things_for_dyck_expression :	
-	( OpDot | OpComma | OpColon | OpSemi | OpAssign | OpAt | OpPound | OpBackTick | OpQuestion | OpUnder)
+	( OpDot | OpComma | OpColon | OpSemi | OpAssign | OpAt | OpPound | OpBackTick | OpQuestion | OpUnder | OpPlus | OpMinus | OpAmp | OpBang | OpTilde | OpGreater | opGreater)
```

Comment:
Are there any other operators that should be in this list?

Reply:
Great question - I tried to name all the operators in the language with `Op` prefix. These are all of them. Swift has general operators as well which are it's own weird token syntax which should get picked up by the production rules under `dyckExcression` with the `literal` and `operator` rules.